package.json

```diff
@@ -14,14 +14,14 @@
         "verify": "yarn lint && yarn test"
     },
     "dependencies": {
-        "@globality/nodule-config": "^2.3.0",
-        "axios": "^0.26.1",
+        "@globality/nodule-config": "^2.15.4-dev.32",
```

Comment:
I will pin when the release version is available.

https://github.com/globality-corp/nodule-config/pull/73